db/prisma/schema.prisma

```diff
@@ -99,7 +99,7 @@ model Post {
 model User {
   id            String    @id @default(cuid())
   name          String? // Check if needed
-  username      String?   @unique
+  username      String   @unique
```

Comment:
username next-auth'da optional olarak verildigi icin oyle eklenmisti. Bu sonradan sorun cikarir optional olarak birakalim PR'in scope'una ait de degil hem